### PR TITLE
use unified attributes

### PR DESCRIPTION
**Pull request by chris-rock**
- use unified attributes
- use messages for only_if

### CODE REVIEW

File: controls/1_2_master_node_scheduler.rb [hunk 1]
```diff
@@ -17,9 +17,11 @@
 
 title '1.2 Master Node: Scheduler'
 
-scheduler = attribute('scheduler', default: kubernetes.processname_scheduler, description: 'The name of the kube scheduler process')
+scheduler = attribute('scheduler')
+# fallback if scheduler attribute is not defined
+kubernetes.scheduler_bin if scheduler.empty?
 
-only_if do
+only_if('scheduled not found') do
```
Comment (rarenerd) [inline controls/1_2_master_node_scheduler.rb]: Typo: scheduled -> scheduler.

File: controls/2_1_worker_node_kubelet.rb [hunk 2]
```diff
@@ -17,9 +17,11 @@
 
 title '2.1 Worker Node: Kubelet'
 
-kubelet = attribute('kubelet', default: kubernetes.processname_kubelet, description: 'The name of the kubelet process')
+kubelet = attribute('kubelet')
+# fallback if kubelet attribute is not defined
+kubelet = kubernetes.kubelet_bin if kubelet.empty?
 
-only_if do
+only_if('kublet not found') do
```
Comment (rarenerd) [inline controls/2_1_worker_node_kubelet.rb]: Typo: kublet -> kubelet.

**Comment by chris-rock** [comment]
Thank you @rarenerd I will fix this early next week.

**Comment by chris-rock** [comment]
@rarenerd issues are fixed and all green now

**Pull request merged**